swebench/harness/log_parsers.py

```diff
@@ -203,6 +203,9 @@ def parse_log_sympy(log: str) -> dict[str, str]:
     for line in log.split("\n"):
         line = line.strip()
         if line.startswith("test_"):
+            if line.endswith("[FAIL]") or line.endswith("[OK]"):
```

Comment:
(could've just done `line.strip().removesuffix("...").removesuffix("...").strip()`, but this should work too)

Reply:
Make sense! Thanks for your review.